pkgs/misc/jitsi-meet-prosody/default.nix

```
@@ -0,0 +1,39 @@
+{ lib, stdenv, fetchurl, dpkg, nixosTests }:
+
+let
+  pname = "jitsi-meet-prosody";
+  version = "1.0.5056";
+  src = fetchurl {
+    url = "https://download.jitsi.org/stable/${pname}_${version}-1_all.deb";
+    sha256 = "06qxa9h2ry92xrk2jklp76nv3sl8nvykdvsqmhn33lz6q6vmw2xr";
+  };
+in
+stdenv.mkDerivation {
+  inherit pname version src;
```

```suggestion
stdenv.mkDerivation rec {
  pname = "jitsi-meet-prosody";
  version = "1.0.5056";

  src = fetchurl {
    url = "https://download.jitsi.org/stable/${pname}_${version}-1_all.deb";
    sha256 = "06qxa9h2ry92xrk2jklp76nv3sl8nvykdvsqmhn33lz6q6vmw2xr";
  };
```
Don't be afraid of rec.

---

I keep forgetting this is a thing... Updated